kebab/src/wrapper.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck because internet is unpredictable
```

Comment:
are you sure typescript will detect the `// @ts-nocheck` now that it's on the second line

Reply:
Yes, it worked. Added because ESLint kept screaming at me.